src/main/java/org/testng/TestListenerAdapter.java

```diff
@@ -25,16 +25,15 @@
  * @author <a href='mailto:the_mindstorm@evolva.ro'>Alexandru Popescu</a>
  */
 public class TestListenerAdapter implements IResultListener2 {
-  private List<ITestNGMethod> m_allTestMethods =
-      Collections.synchronizedList(Lists.<ITestNGMethod>newArrayList());
-  private List<ITestResult> m_passedTests = Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestResult> m_failedTests = Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestResult> m_skippedTests = Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestResult> m_failedButWSPerTests = Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestContext> m_testContexts= Collections.synchronizedList(new ArrayList<ITestContext>());
-  private List<ITestResult> m_failedConfs= Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestResult> m_skippedConfs= Collections.synchronizedList(Lists.<ITestResult>newArrayList());
-  private List<ITestResult> m_passedConfs= Collections.synchronizedList(Lists.<ITestResult>newArrayList());
+  private Collection<ITestNGMethod> m_allTestMethods = new ConcurrentLinkedQueue<>();
+  private Collection<ITestResult> m_passedTests = new ConcurrentLinkedQueue<>();
+  private Collection<ITestResult> m_failedTests = new ConcurrentLinkedQueue<>();
+  private Collection<ITestResult> m_skippedTests = new ConcurrentLinkedQueue<>();
+  private Collection<ITestResult> m_failedButWSPerTests = new ConcurrentLinkedQueue<>();
+  private Collection<ITestContext> m_testContexts= new ConcurrentLinkedQueue<>();
```

Comment:
could you fix the missing spaces at the same time?

Reply:
amended the commit, fixed this and pushed in the changes.